nntrainer/compiler/tflite_opnode.cpp

```diff
@@ -111,12 +112,33 @@ void TfOpNode::setLayerNode(const LayerNode &layer) {
       return &t;
     },
     context.getNumWeights());
+
+  if (context.getNumWeights() == 0) {
+    is_trainable = false;
+  }
 }
 
 void TfOpNode::setWeightTransformFn(TransformFn fn) { weight_transform = fn; }
 
 void TfOpNode::setInputTransformFn(TransformFn fn) { input_transform = fn; }
 
+void TfOpNode::setWeights(Variables weights_) {
+  unsigned int cnt = 0;
+  for (auto &w : weights_) {
+    const unsigned int UNIT = w->batch();
+    const unsigned int CHANNEL = w->channel();
+    const unsigned int HEIGHT = w->height();
+    const unsigned int WIDTH = w->width();
+
+    auto weight_data = weights.at(cnt)->getData();
+    auto *ptr = const_cast<float *>(weight_data);
+    memcpy(&ptr[0], &w->getData()[0],
```

Comment:
how about using tensor op rather than using memcpy?

Reply:
can i use ```weights.at(cnt)->putData(weights_)```?

Reply:
you can use https://github.com/nnstreamer/nntrainer/blob/65d5f04ab8cf666d4ec784d663a949103035d59b/nntrainer/tensor/tensor.h#L1055 
or 
https://github.com/nnstreamer/nntrainer/blob/65d5f04ab8cf666d4ec784d663a949103035d59b/nntrainer/tensor/tensor.h#L1061